recipes/eternafold/build.sh

```diff
@@ -0,0 +1,25 @@
+# Move to conda-specific src directory location
+cd $SRC_DIR/src
+
+# Build Eternafold
+make CXX=$CXX
+
+# Move built binaries to environment-specific location
+mkdir -p -v $PREFIX/bin/eternafold-bin
+cp contrafold api_test score_prediction $PREFIX/bin/eternafold-bin
+
+# Move relevant repo files to lib folder
+cd $PREFIX
+mkdir -p -v $PREFIX/lib/eternafold-lib
+cp -R $SRC_DIR/* $PREFIX/lib/eternafold-lib
```

Comment:
Looks like you're installing an entire copy of the source into the package. This includes copies of the binaries which were already installed to /bin/eternafold-bin, and files that are added during conda during the build process!

Please do not install unneeded files. I doubt that the copies of binaries are needed, and this doesn't seem to be a development package, so the entire source code is also not needed.

Be more specific about which files you need to copy.